### PR TITLE
Adapt automatic screenshots for contact diary extension (EXPOSUREAPP-5115)

### PR DESCRIPTION
**Pull request by LukasLechnerDev**
I adapted our screenshots for the contact diary extension to look like our figma designs:

image
image
image
image

### CODE REVIEW

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt [hunk 1]
```diff
@@ -18,40 +19,35 @@ object DiaryData {
...
-                R.string.contact_diary_person_encounter_mask_with,
-                R.string.contact_diary_person_encounter_environment_inside
-            ),
-            "Notizen notizen",
-            ListItem.Type.PERSON
+            "Rewe",
+            Duration.standardMinutes(30),
+            attributes = null,
+            circumstances = null,
+            ListItem.Type.LOCATION
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt:26]: On the screenshot, the icon for a person is shown instead. please double check.

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt [hunk 2]
```diff
@@ -18,40 +19,35 @@ object DiaryData {
...
-                R.string.contact_diary_person_encounter_environment_inside
+                R.string.contact_diary_person_encounter_duration_below_15_min,
+                R.string.contact_diary_person_encounter_environment_outside
             ),
-            "Notizen notizen",
+            null,
             ListItem.Type.PERSON
         ),
 
         ListItem.Data(
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt:40]: UX would like us to remove "Bäcker" entirely

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt (hunk 1 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt:26]: Good eyes 👀 😀  => I changed the icon, see new screenshot: 

ContactDiaryOverviewFragment

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt (hunk 2 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/DiaryData.kt]: Done, see new screenshot: 

ContactDiaryOverviewFragment_2

**Review by ralfgehrer — APPROVED** [review]

**Review by d4rken — APPROVED** [review]

**Pull request merged**